client/CBitmapHandler.cpp

```diff
@@ -109,43 +112,43 @@ SDL_Surface * BitmapHandler::loadBitmapFromDir(std::string path, std::string fna
 		logGlobal->warnStream() << "Call to loadBitmap with void fname!";
 		return nullptr;
 	}
-	if (!CResourceHandler::get()->existsResource(ResourceID(path + fname, EResType::IMAGE)))
+	if(!CResourceHandler::get()->existsResource(ResourceID(path + fname, EResType::IMAGE)))
 	{
 		return nullptr;
 	}
 
-	SDL_Surface * ret=nullptr;
+	SDL_Surface * ret = nullptr;
 
 	auto readFile = CResourceHandler::get()->load(ResourceID(path + fname, EResType::IMAGE))->readAll();
 
-	if (isPCX(readFile.first.get()))
-	{//H3-style PCX
+	if(isPCX(readFile.first.get()))
+	{ //H3-style PCX
 		ret = loadH3PCX(readFile.first.get(), readFile.second);
-		if (ret)
+		if(ret)
 		{
-			if(ret->format->BytesPerPixel == 1  &&  setKey)
+			if(ret->format->BytesPerPixel == 1 && setKey)
 			{
-				CSDL_Ext::setColorKey(ret,ret->format->palette->colors[0]);
+				CSDL_Ext::setColorKey(ret, ret->format->palette->colors[0]);
 			}
 		}
 		else
 		{
-			logGlobal->errorStream()<<"Failed to open "<<fname<<" as H3 PCX!";
+			logGlobal->errorStream() << "Failed to open " << fname << " as H3 PCX!";
 			return nullptr;
 		}
 	}
 	else
 	{ //loading via SDL_Image
 		ret = IMG_Load_RW(
-		          //create SDL_RW with our data (will be deleted by SDL)
-		          SDL_RWFromConstMem((void*)readFile.first.get(), readFile.second),
-		          1); // mark it for auto-deleting
-		if (ret)
+			//create SDL_RW with our data (will be deleted by SDL)
+			SDL_RWFromConstMem((void *)readFile.first.get(), readFile.second),

```

Comment:
Not sure what to think about spacebar in casting, Is this side-effect work of uncrustify?